e2e/tests/session_steps_test.go

```diff
@@ -297,3 +250,71 @@ func (s *suite) waitForNewBlockEvent(
 		s.Log("Success; message detected before timeout.")
 	}
 }
+
```

Comment:
These helpers LGTM. Did you take some time to make sure we reuse them elsewhere? (i.e. some of the work I did that should leverage them)

Reply:
Yessir! :saluting_face: My observation was that the spaghetti code was strewn between the session, relay, and settlement scenarios.